packages/flutter_test/lib/src/controller.dart

```diff
@@ -975,6 +975,9 @@ abstract class WidgetController {
   /// that will build child specified by `finder` when there are multiple
   ///[Scrollable]s.
   ///
+  /// Scroll is performed until the start of the `finder` is visible. This is
+  /// due to the default parameter values of [Scrollable.ensureVisible] method.
```

Comment:
The second sentence sounds like an implementation detail that doesn't really belong in a doc? If this is a problem we could wire up a parameters passed in here to Scrollable.ensureVisible, though.

Reply:
I thought of wiring up the parameter but, as you say, there's currently no problem that necessitates the use of that parameter. Just that the behavior is not clear from documentation. I also did not want to just say: "until the start of `finder` is visible` and leave it at that because this method does not own that functionality. Therefore, I wanted to provide a link to how the behavior is determined so someone else, if needed, can make a request to plumb that parameter.